### DISCUSSION TITLE
Unable to Update Pychron

### DISCUSSION THREAD

**Discussion by gennachiaro** [NMGRL/pychron#2164] [Q&A] [answered]
I am unable to update my Pychron.  I have it set to check for updates at startup, but there is not an update repo listed in preferences, so maybe that is causing the problem?

![image](https://user-images.githubusercontent.com/56849646/163039452-46812d70-fd5c-4095-87a2-647f1cf14ae9.png)

I tried to manually check for updates via Help > Check for Updates and the terminal says it is checking but there is no progress.

I've attached the log file here: 
[pychron.current.log](https://github.com/NMGRL/pychron/files/8475969/pychron.current.log)

Thanks for your help!

**Comment by jirhiker** [comment, marked as answer]
You need to set the Update Repo *Name:*

use ```NMGRL/pychron```

**Reply by gennachiaro** [reply]
Ok great, thank you!  This works and my pychron updated to the newest version.

**Comment by jirhiker** [comment]
Another options is to update pychron manually.

open a terminal and cd to the pychron source directory

```
cd ~/.pychron.0/pychron
```

use git to pull updates
```
git pull
```
